WooCommerce/Classes/ViewRelated/Orders/Fulfillment/Cells/PickListTableViewCell.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+import Gridicons
+import Yosemite
+
+
+/// Pick List: Renders a row that displays a single Product.
+///
+class PickListTableViewCell: UITableViewCell {
+
+    /// ImageView
+    ///
+    @IBOutlet private var productImageView: UIImageView!
+
+    /// Label: Name
+    ///
+    @IBOutlet private var nameLabel: UILabel!
+
+    /// Label: Quantity
+    ///
+    @IBOutlet private var quantityLabel: UILabel!
+
+    /// Label: SKU
+    ///
+    @IBOutlet private var skuLabel: UILabel!
+
+    /// Product Name
+    ///
+    var name: String? {
+        get {
+            return nameLabel?.text
+        }
+        set {
+            nameLabel?.text = newValue
+        }
+    }
+
+    /// Number of Items
+    ///
+    var quantity: String? {
+        get {
+            return quantityLabel?.text
+        }
+        set {
+            quantityLabel?.text = newValue
+        }
+    }
+
+    /// Item's SKU
+    ///
+    var sku: String? {
+        get {
+            return skuLabel?.text
+        }
+        set {
+            skuLabel?.text = newValue
+        }
+    }
+
+    // MARK: - Overridden Methods
+
+    required init?(coder aDecoder: NSCoder) {
+        // initializers don't call property observers,
+        // so don't set the default for mode here.
+        super.init(coder: aDecoder)
+    }
+
+    class func makeFromNib() -> PickListTableViewCell {
+        return Bundle.main.loadNibNamed("ProductDetailsTableViewCell", owner: self, options: nil)?.first as! PickListTableViewCell
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        productImageView.image = Gridicon.iconOfType(.product)
+        productImageView.tintColor = StyleManager.wooGreyBorder
+        selectionStyle = .none
+
+        nameLabel.applyBodyStyle()
+        quantityLabel.applyBodyStyle()
+        skuLabel.applySecondaryFootnoteStyle()
+
+        nameLabel?.text = ""
+        quantityLabel?.text = ""
+        skuLabel?.text = ""
+    }
+}
+
+
+// MARK: - Public Methods
+//
+extension PickListTableViewCell {
+    func configure(item: OrderItemViewModel, image: UIImage? = nil) {
```

Comment:
I know this is beyond the scope of this PR, but this method signature is the same as a method in ProductDetailsTableViewCell. 

I have been thinking for some time that we might benefit from an approach were we invert the dependencies a little bit in our datasources, following precisely this pattern: instead of assign values to outlets in the cellForRowAtIndexPath method, pass a model object to the cell and let the cell populate itself.

If we made all the cells following this pattern implement a common interface (which is what you have already done here implicitly), generic on a type T, we could get rid of a lot of boilerplate with a datasource also generic on a type T and a second type CellThatImplementsConfigure<T>

So, leaving my digression aside for now, I think we should keep an eye on this, and if we see this abstraction piping up again (the abstraction being a cell with a configure() method generic on a model object type) we might want to formalise it in a protocol?

Reply:
Yes I think that would be nice! Especially for the cells that we know are specialized to one screen and won't get re-used. I like the public property approach (cell.titleLabel.text = "foo") when I know we're going to re-use the cell in a lot of places.

Reply:
I'll need your help with that though, when you find that opportunity again. I can recognize the pattern but I've never written generics before.